bot/cogs/moderation/management.py

```diff
@@ -257,7 +257,11 @@ def infraction_to_string(self, infraction: utils.Infraction) -> str:
     # This cannot be static (must have a __func__ attribute).
     def cog_check(self, ctx: Context) -> bool:
         """Only allow moderators to invoke the commands in this cog."""
```

Comment:
This docstring needs to be updated to reflect the new check.

Reply:
Thanks! will do the changes.

Reply:
Updated the doc-string. I am not familiar with terminology yet so, not sure if doc-string makes much sense.